idb/postgres/postgres_migrations.go

```diff
@@ -19,6 +19,14 @@ import (
 	"github.com/algorand/indexer/idb/postgres/internal/types"
 )
 
+// AppBoxMigration is used to interpolate the field AppBoxMigration of `cmd/texttosource/main.go` in the SQL template `idb/postgres/internal/schema/setup_postgres.sql`
+const AppBoxMigration = `CREATE TABLE IF NOT EXISTS app_box (
```

Comment:
@shiqizng - in order to avoid duplicated `CREATE TABLE` logic, I'm storing the SQL in `AppBoxMigration` and populating it both in the migration step, as well as in the SQL generation script using a a template.

Reply:
We deprecate the migrations pretty aggressively, only guaranteeing a 2 week grace period. So duplicate code here is not really a problem.

Let's keep the simple .sql file without template processing, and we'll be able to remove the second definition in the N + 1 release.

[Here is the policy if you are curious for more details.](https://docs.google.com/document/d/1ZQjx_gAO9uhqSy25m00puPHxe6CmQ8y-RVkamOTTDbg/edit#heading=h.djearkgog5ci)

Reply:
I'll revert to the original approach (copy/pasta the CREATE TABLE). Meanwhile, I'd love to be able to read the policy and have requested access.

Reply:
This has been reverted @winder